test-configs.yaml

```diff
@@ -1566,7 +1940,41 @@ test_configs:
   - device_type: bcm2836-rpi-2-b
     test_plans:
       - baseline
+      - ltp-admin_tools
```

Comment:
For test configs, we can easily have a YAML anchor as we can have multiple test config entries with the same device type.  For example:
```yaml
ltp-test-plans: &ltp_test_plans
  - ltp-admin_tools
  - ltp-connectors
  - ltp-containers
# etc...

test_configs:
  - device_type: bcm2836-rpi-2-b
    test_plans:
      - baseline
      - sleep
      - usb

  - device_type: bcm2836-rpi-2-b
    test_plans: *ltp_test_plans
```

Reply:
Also we'll see how things go when running everything on all the test devices, maybe we'll want to also have anchors with some LTP subsets.  For example, `ltp-fs-plans`, `ltp-mem-plans` etc... with just a handful of things to run in order to optimise how labs' capacity is being used.  But that's for a bit later, once we have started collecting results.  It's probably going to be an on-going process in fact, with new labs and new devices and new versions of LTP.

Reply:
> For test configs, we can easily have a YAML anchor as we can have multiple test config entries with the same device type. For example:
> 
> ```yaml
> ltp-test-plans: &ltp_test_plans
>   - ltp-admin_tools
>   - ltp-connectors
>   - ltp-containers
> # etc...
> 
> test_configs:
>   - device_type: bcm2836-rpi-2-b
>     test_plans:
>       - baseline
>       - sleep
>       - usb
> 
>   - device_type: bcm2836-rpi-2-b
>     test_plans: *ltp_test_plans
> ```

updated the patch with this change.